docs/reference/files/charmcraft-yaml-file.rst

```diff
@@ -29,7 +29,7 @@ to declare information about the project in a form that can be used by Charmcraf
 .. note::
 
     If you're starting from an empty file, the only required key is the ``type`` key.
-    However, depending on what value you set it to (``charm`` or ``bundle``), other
```

Comment:
If `type: bundle`, the charm will simply fail to pack, right? No hint to switch to `charm`?

Reply:
Correct. For Charmcraft 4.0 the only valid `type` is `charm`. (And switching to `charm` from a bundle would fail anyway since they're quite different things.)

Reply:
Actually I amend that: if you set `type: bundle` in `charmcraft.yaml` and run `charmcraft pack`, it will tell you that bundle support has been removed and to use charmcraft 3.